### PR TITLE
Add initial version of statistics pingback

### PR DESCRIPTION
**Pull request by jamalex**
### Summary

Spec of data format here:
https://docs.google.com/document/d/14oZ7RHkPzRJgcALd6B6iwz1RDT_kQ03omdClso0pSno/edit#heading=h.fv461r5wfbey

### Reviewer guidance

Run the following (with a bunch of data and content in your database) to test the pingback:
`kolibri --debug manage ping --server http://nutritionfacts-staging.nanoapp.io`
(it will also print out the data being sent)

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by jamalex** [comment]
The server side of this is in this commit:
https://github.com/learningequality/nutritionfacts/pull/3/commits/0c7fda9419e5c40bbe2280a904966e5d1e1aac91

**Comment by jamalex** [comment]
Next step: @ralphiee22 offered to help write some tests for this code before we merge it in. THANK YOUUUUU!

File: kolibri/core/analytics/utils.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    except:  # noqa
+        pass
+    return jsondata
+
+
+def extract_facility_statistics(facility):
+
+    dataset_id = facility.dataset_id
+
+    learners = FacilityUser.objects.filter(dataset_id=dataset_id).exclude(roles__kind__in=[role_kinds.ADMIN, role_kinds.COACH])
```
Comment (ralphiee22) [inline kolibri/core/analytics/utils.py:53]: We don't exclude users who only have the superuser permission enabled.

File: kolibri/core/analytics/utils.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    coaches = FacilityUser.objects.filter(dataset_id=dataset_id, roles__kind__in=[role_kinds.ADMIN, role_kinds.COACH])
+
+    usersessions = UserSessionLog.objects.filter(dataset_id=dataset_id)
+    contsessions = ContentSessionLog.objects.filter(dataset_id=dataset_id, time_spent__lt=3600 * 2)
+
+    usersess_agg = (usersessions
```
Comment (ralphiee22) [inline kolibri/core/analytics/utils.py:60]: perhaps a comment explaining that the aggregates below are used to calculate the first time this device has been interacted with and the last time it has been interacted with?

**Review by ralphiee22 — APPROVED** [review]
Manually tested 👍 
LGTM but tests to follow in separate PR.

**Comment by rtibbles** [comment]
Will hold off on merging, as not vital for first round of integration testing.

File: kolibri/core/analytics/utils.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        "pc": [item["count"] for item in pop],  # popular_counts
+        "s": (localfiles.aggregate(Sum("file_size"))["file_size__sum"] or 0) / (2 ** 20),  # storage
+        "ss": summarylogs.count(),  # summ_started
+        "sc": summarylogs.exclude(completion_timestamp=None).count(),  # summ_complete
+        "sk": sesslogs_by_kind,  # sess_kinds
+        "suc": contsessions_user.count(),  # sess_user_count
+        "sac": contsessions_anon.count(),  # sess_anon_count
+        "sut": int((contsessions_user.aggregate(total_time=Sum("time_spent"))["total_time"] or 0) / 60),  # sess_user_time
+        "sat": int((contsessions_anon.aggregate(total_time=Sum("time_spent"))["total_time"] or 0) / 60),  # sess_anon_time
+    }
```
Comment (indirectlylit) [inline kolibri/core/analytics/utils.py:168]: Maybe move the comments above the items being returned to make it easier to scan?

File: kolibri/core/analytics/utils.py (hunk 1 above)
Comment (jamalex) [reply kolibri/core/analytics/utils.py:53]: Decision: edge case, ignoring

**Comment by jamalex** [comment]
Updated to address comments, and added timezones/facility settings.

**Pull request merged**